packages/babel-parser/test/fixtures/flow/comment/01-type-include/output.json

```diff
@@ -64,85 +64,18 @@
         "body": {
           "type": "ClassBody",
           "start": 14,
-          "end": 41,
+          "end": 44,
           "loc": {
             "start": {
               "line": 1,
               "column": 14
             },
             "end": {
-              "line": 3,
-              "column": 1
+              "line": 2,
+              "column": 28
             }
           },
-          "body": [
-            {
-              "type": "ClassProperty",
-              "start": 23,
-              "end": 36,
-              "loc": {
-                "start": {
-                  "line": 2,
-                  "column": 7
-                },
-                "end": {
-                  "line": 2,
-                  "column": 20
-                }
-              },
-              "static": false,
-              "key": {
-                "type": "Identifier",
-                "start": 23,
-                "end": 27,
-                "loc": {
-                  "start": {
-                    "line": 2,
-                    "column": 7
-                  },
-                  "end": {
-                    "line": 2,
-                    "column": 11
-                  },
-                  "identifierName": "prop"
-                },
-                "name": "prop"
-              },
-              "computed": false,
-              "variance": null,
-              "typeAnnotation": {
-                "type": "TypeAnnotation",
-                "start": 27,
-                "end": 35,
-                "loc": {
-                  "start": {
-                    "line": 2,
-                    "column": 11
-                  },
-                  "end": {
-                    "line": 2,
-                    "column": 19
-                  }
-                },
-                "typeAnnotation": {
-                  "type": "StringTypeAnnotation",
-                  "start": 29,
-                  "end": 35,
-                  "loc": {
-                    "start": {
-                      "line": 2,
-                      "column": 13
-                    },
-                    "end": {
-                      "line": 2,
-                      "column": 19
-                    }
-                  }
-                }
-              },
-              "value": null
-            }
-          ]
```

Comment:
This is wrong.

Reply:
This comment is pretty vague. Can you please be more specific and explain exactly what's wrong with the code

Reply:
Fixed in b0588ae2c9c50ca5502e920a19b41644215d1712